heron/packing/src/java/com/twitter/heron/packing/roundrobin/RoundRobinPacking.java

```diff
@@ -96,212 +311,94 @@ public double getContainerCpuHint(Map<String, List<String>> packing) {
   /**
    * Provide disk per container.
    *
-   * @param packing packing output.
+   * @param allocation packing output.
    * @return disk per container.
    */
-  public long getContainerDiskHint(Map<String, List<String>> packing) {
+  protected long getContainerDiskHint(Map<String, List<String>> allocation) {
     long defaultContainerDisk =
-        instanceDiskDefault * getLargestContainerSize(packing) + DEFAULT_DISK_PADDING;
+        instanceDiskDefault * getLargestContainerSize(allocation)
+            + DEFAULT_DISK_PADDING_PER_CONTAINER;
 
     List<TopologyAPI.Config.KeyValue> topologyConfig = topology.getTopologyConfig().getKvsList();
 
     String diskHint = TopologyUtils.getConfigWithDefault(
         topologyConfig, com.twitter.heron.api.Config.TOPOLOGY_CONTAINER_DISK_REQUESTED,
-            Long.toString(defaultContainerDisk));
+        Long.toString(defaultContainerDisk));
 
     return Long.parseLong(diskHint);
   }
 
   /**
-   * Provide default ram for instances of component whose Ram requirement is not specified.
+   * Provide ram per container.
    *
-   * @return default ram in bytes.
+   * @param allocation packing
+   * @return Container ram requirement
    */
-  public long getDefaultInstanceRam(Map<String, List<String>> packing) {
+  public long getContainerRamHint(Map<String, List<String>> allocation) {
```

Comment:
If it's dead code without an immediate resurrection in view, I'd say we remove it.